src/Impostor.Server/Events/MultiDisposable.cs

```diff
@@ -19,8 +19,8 @@ public void Dispose()
         {
             foreach (var disposable in _disposables)
             {
-                disposable.Dispose();
+                disposable?.Dispose();
```

Comment:
This line is throwing NullReferenceExceptions in other cases, unrelated to shutting down the server. In my opinion, this change introducing the null conditional operator should be merged regardless if this pr is merged or not.